app/component-library/components/Tags/TagUrl/TagUrl.stories.tsx

```diff
@@ -1,30 +1,38 @@
-/* eslint-disable react-native/no-inline-styles, no-console */
-
-// Third party dependencies.
+/* eslint-disable no-console */
+/* eslint-disable react-native/no-inline-styles */
+/* eslint-disable react/display-name */
+// External dependencies.
 import React from 'react';
 import { View } from 'react-native';
-import { storiesOf } from '@storybook/react-native';
 
 // Internal dependencies.
-import TagUrl from './TagUrl';
-import { TEST_IMAGE_SOURCE, TEST_LABEL } from './TagUrl.constants';
+import { default as TagUrlComponent } from './TagUrl';
+import { SAMPLE_TAGURL_PROPS } from './TagUrl.constants';
+
+const TagUrlMeta = {
+  title: 'Component Library / Tags',
+  component: TagUrlComponent,
+  argTypes: {
+    label: {
+      control: { type: 'text' },
+      defaultValue: SAMPLE_TAGURL_PROPS.label,
+    },
+    ctaLabel: {
```

Comment:
I think the controls should consistently reflect the props of the component but I see that the `cta` is an object. Could we use dot notation for args that are keys for an object prop?
```suggestion
   'cta.label': {
```
![Screenshot 2023-10-24 at 1 07 15 PM](https://github.com/MetaMask/metamask-mobile/assets/8112138/3ea23a23-b216-4873-bd57-233fe19f0d67)